### DISCUSSION TITLE
Cypress not accessing multimedia API calls

### DISCUSSION THREAD

**Discussion by bansal-shivankar-tft** [cypress-io/cypress#20482] [Questions and Help]
I have an API having .mp3 as an extension but cypress is not able to access that API and giving 404 but when I am manually executing that scenario in chrome browser it is working fine for me.
Please let me know any workaround for this

**Comment by portyu9** [comment]
Hi @bansal-shivankar-tft 

Cypress does not block `.mp3` requests.

A `404` means the URL reaching the server is different or is being resolved differently during the Cypress run.

First inspect the exact media request:

```js
cy.intercept('GET', '**/*.mp3*').as('audio')

cy.wait('@audio').then(({ request, response }) => {
  expect(response.statusCode).to.eq(200)
  cy.log(request.url)
})
```

Compare that URL, query string, authentication headers/cookies, and `Referer` with the successful manual Chrome request.

If you are trying to test the MP3 endpoint directly, use:

```js
cy.request({
  url: 'https://example.com/audio/file.mp3',
  encoding: 'binary',
}).its('status').should('eq', 200)
```

So the fix is not a special multimedia setting in Cypress—the server route/authentication for the `.mp3` request needs to be made identical to the working browser request.

**Reply by portyu9** [reply]
@bansal-shivankar-tft Please mark this answer as accepted if it helped, thank you.
